src/sif/autocost.cc

```diff
@@ -65,11 +65,17 @@ constexpr float kLeftSideTurnCosts[] = {kTCStraight,         kTCSlight,  kTCUnfa
 constexpr float kMinFactor = 0.1f;
 constexpr float kMaxFactor = 100000.0f;
 
+// Default auto attributes
+constexpr float kDefaultAutoHeight = 1.6f; // Meters (62.9921 inches)
+constexpr float kDefaultAutoWidth = 1.4f;  // Meters (55.1181 inches)
```

Comment:
this should be the original value of 1.9. 1.4 is incredibly narrow!

Reply:
Okey, fixed